WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1911,6 +1931,144 @@ static GSourceFuncs _handlerIntervention =
             return _page;
         }
 #endif
+
+        std::vector<std::string> splitString(const std::string &s, char delim)
+        {
+            std::vector<std::string> elems;
+            std::stringstream ss(s);
+            std::string item;
+            while (std::getline(ss, item, delim)) {
+                elems.push_back(std::move(item));
+            }
+            return elems;
+        }
+
+        bool readStatmLine(pid_t pid, std::string &statmLine)
```

Comment:
readStatmLine and parseRssFromStatmLine are used only in collectMetricsOnLoadStart. I think you can move them to collectMetricsOnLoadStart as lambdas or move them outside of WebKitImplementation class.